lang_python.adoc

```diff
@@ -13,130 +13,155 @@ To generate Python modules, Kaitai Struct compiler should be invoked with
+Reference: https://packages.debian.org/testing/python3-kaitaistruct[image:https://repology.org/badge/version-for-repo/debian_testing/python:python-kaitaistruct.svg[package info for Debian]], https://packages.ubuntu.com/focal/python3-kaitaistruct[image:https://repology.org/badge/version-for-repo/ubuntu_20_04/python:python-kaitaistruct.svg[package info for Ubuntu]]
 
-** For Python 3.x:
+** For Python 2.x:
 +
 [source,shell]
-sudo apt-get install python3-kaitaistruct
+sudo apt-get install -y python-kaitaistruct
 +
-Reference: https://packages.debian.org/buster/python3-kaitaistruct[package info for Debian], https://packages.ubuntu.com/eoan/python3-kaitaistruct[package info for Ubuntu]
+Reference: https://packages.debian.org/stable/python-kaitaistruct[image:https://repology.org/badge/version-for-repo/debian_stable/python:python-kaitaistruct.svg[package info for Debian]], https://packages.ubuntu.com/focal/python-kaitaistruct[image:https://repology.org/badge/version-for-repo/ubuntu_20_04/python:python-kaitaistruct.svg[package info for Ubuntu]]
+
 
-* If your project is using *requirements.txt* to manage your
-  dependencies, add the following line to it:
+* Add the following dependency specifier into the package configuration
+of the packaging tool you use:
 +
 ----
 kaitaistruct
 ----
 +
-and then run `pip install -r requirements.txt` to update all your
-dependencies.
+and then run `python3 -m pip install --upgrade -e ./` in the dir with your project
+in order to install your package and all its dependencies in
+an editable way. If you don't want to package your software,
+but only install its dependencies, add `kaitaistruct` into `requirements.txt`.
 
-* Otherwise, you can just install the package manually using *pip*
+* Otherwise, you can just install the package manually using `pip`
   (Python package manager):
 +
 [source,shell]
-pip install kaitaistruct
+python3 -m pip install --upgrade kaitaistruct
 
-[[add-runtime-snapshot]]
+[[install-runtime-snapshot]]
 ==== Using latest (snapshot) version
 
-If you're using latest (unreleased AKA "unstable" AKA "snapshot") build
-of Kaitai Struct compiler, that will require latest runtime libraries as
-well:
+If you're using latest (unreleased aka "bleeding edge" aka "unstable"
+aka "snapshot" aka "nightly") build of Kaitai Struct compiler, that will
+require latest runtime libraries as well:
 
-* If your project is using *requirements.txt* to manage your
-  dependencies, add the following line to it:
+* You can a make `pip` to use a snapshot version of the runtime by
+  adding a https://www.python.org/dev/peps/pep-0508/[PEP 508]
+  dependency specifier into the config of your favorite build tool:
 +
 ----
 kaitaistruct @ git+https://github.com/kaitai-io/kaitai_struct_python_runtime.git
 ----
 +
-and then run `pip install -r requirements.txt` to update all your
-dependencies.
+.
```

Comment:
I'd really rather remove this dot, as a lone dot on a separate line looks quite weird when you display the rendered Asciidoctor output. On top of that, you aren't adding the full stop after the code block like this anywhere else in the document.

Reply:
Done.